src/components/Calculator.jsx

```diff
@@ -7,120 +7,124 @@ import {
 } from 'react-bootstrap';
 import calc from '../calc-events/calc';
 
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      results: null,
-      next: null,
-      operation: null,
-      valueToShow: 0,
-    };
-    this.clickHandler = this.clickHandler.bind(this);
-  }
+function Calculator() {
```

Comment:
- [ ] (Optional) Consider converting this function to `Es6` function. 👍

Reply:
Will also work on that suggestion in the continuity of the project... thanks for the suggestion.